recipes/xorg-proto/all/conanfile.py

```diff
@@ -40,7 +39,7 @@ def build_requirements(self):
         self.build_requires("automake/1.16.3")
         self.build_requires("xorg-macros/1.19.3")
         self.build_requires("pkgconf/1.7.4")
-        if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
+        if self._settings_build.os == "Windows" and not os.environ["CONAN_BASH_PATH"]:
```

Comment:
🤔 should be a conf check now AFAIK

Reply:
https://github.com/conan-io/conan-center-index/blob/044e188e95c82b9ff0e9bb71fdc9e1c9618b15c7/recipes/gumbo-parser/all/conanfile.py#L55-L58